pom.xml

```diff
@@ -205,6 +206,20 @@
 			<artifactId>test-container</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+
+		<!-- Transitive dependency version overrides for Vulnerabilities: -->
+		<!-- overriding version commons-io for vertex 3.9.3 - Vulnerability -->
+		<dependency>
+			<groupId>commons-io</groupId>
+			<artifactId>commons-io</artifactId>
+			<version>2.5</version>
+		</dependency>
```

Comment:
should we open an issue to Vert.x raising the vulnerability they have and tracking the issue here?